Lib/asyncio/events.py

```diff
@@ -20,6 +20,10 @@
 from . import format_helpers
 
 
+class SendfileUnsupportedError(ValueError):
```

Comment:
`SendFileNotSupported`

Why not `raise NotImplementedError('sendfile is not available')`?